mscore/qml/palettes/Palette.qml

```diff
@@ -767,7 +767,7 @@ GridView {
                 // force not hiding palette cell if it is being dragged to a score
                 enabled: paletteCell.paletteDrag
                 target: mscore
-                onElementDraggedToScoreView: paletteCell.paletteDrag = false
+                function onElementDraggedToScoreView() { paletteCell.paletteDrag = false; }
```

Comment:
Really?

I’d have thought this…

    onElementDraggedToScoreView: function () { paletteCell.paletteDrag = false; }

But if it works… sure.

Reply:
It does for me

Reply:
I don’t doubt that ☻ it’s just that JavaScript continues to irritate me, the more I learn about it ☺